.github/workflows/build.yml

```diff
@@ -0,0 +1,35 @@
+name: Build
+
+on:
+  push:
+    branches: [ "master" ]
+  pull_request:
+    branches: [ "master" ]
+
+jobs:
+  build:
+    runs-on: windows-latest
+
+    steps:
+    - uses: actions/checkout@v3
+
+    - name: Add msbuild to PATH
+      uses: microsoft/setup-msbuild@v1.1
+
+    - name: Restore
+      run: msbuild pbench.sln /t:restore /p:Configuration=Release /p:Platform=x86
+
+    - name: Build
+      run: msbuild pbench.sln /t:build /p:Configuration=Release /p:Platform=x86 /p:RestorePackages=false
+
+    - name: Prepare artifact
```

Comment:
'Prepare artifact**s**'

Reply:
I think 'Prepare artifact' is correct because there is only one artifact but with two files.

Reply:
ok